data/money8203.json

```diff
@@ -34,6 +34,12 @@
       "url": "https://twitter.com/manishtyagi__",
       "icon": "FaTwitter"
     },
+    {
+      "group": "Socials",
+      "name": "Let's Thread together",
+      "url": "https://threads.net/@manishtyagi__",
+      "icon": "FaThreads"
```

Comment:
`FaThreads` is not a valid icon. Please update it with something from [these icons](https://linkfree.io/icons).

Reply:
`FaInstagram` might be the closest replacement for this icon key?